traffic_ops/app/lib/API/DeliveryServiceServer.pm

```diff
@@ -27,6 +28,11 @@ sub index {
 	my @data;
 	my $orderby = $self->param('orderby') || "deliveryservice";
 
+	#FOR THE REVIEWER - Currently I do not check DS tenancy here.
```

Comment:
As I saw it, the ds-admin should not be aware of specific servers existence, so this API should not be available to him.
However, as you see usecases a ds-admin may approach this API, I'll had the tenancy check